README.rst

```diff
@@ -62,6 +62,12 @@ Debian family (Ubuntu...)
 ::
 
   sudo apt-get install build-essential zlib1g-dev libbz2-dev libssl-dev libreadline-dev libncurses5-dev libsqlite3-dev libgdbm-dev libdb-dev libexpat-dev libpcap-dev liblzma-dev libpcre3-dev
+  if errors on compiling last version of cpython do :
```

Comment:
Can you please leave an empty line and use proper punctuation? Thanks!

Reply:
I think appending libffi-dev to the package list is sufficient